src/material-experimental/mdc-table/table.scss

```diff
@@ -27,3 +27,18 @@
 mat-row.mat-mdc-row, mat-header-row.mat-mdc-header-row, mat-footer-row.mat-mdc-footer-row {
   border-bottom: none;
 }
+
+// Cells need to inherit their background in order to overlap each other when sticky.
+// The background needs to be inherited from the table, tbody/tfoot, row
+// (already set in MDC), and cell.
+.mat-mdc-table tbody, .mat-mdc-table tfoot,
+.mat-mdc-cell, .mat-mdc-footer-cell {
+  background: inherit;
```

Comment:
Instead of having these inherit the background, would it make sense to just assign the background to the cells in the theme? This feels like something that we could forget about if we remove the background from the theme.

Reply:
Do you mean assign the card background to the cells? The benefit with the `inherit` approach is that users can style the table host element with a background and it works as expected, rather than identifying that they need to style the cell background.

Reply:
Alright, makes sense. Let's keep it like this.